packages/gatsby-theme-vtex/src/components/Shelf/index.tsx

```diff
@@ -45,9 +47,11 @@ const Shelf: FC<Props> = ({
     autoplay,
   })
 
+  showArrows = showArrows && products.length >= Math.max(...pageSizes)
+
   return (
     <Fragment>
-      {title ? <ShelfTitle title={title} /> : null}
+      {title ? <ShelfTitle title={title} variant={titleVariant} /> : null}
```

Comment:
why is it called titleVariant and not only variant ? I think the best way of using variants in here is creating a variable called `variant` and passing `${variant}.title` as the `<ShelfTitle />` component variant